components/compositing/compositor.rs

```diff
@@ -642,6 +662,10 @@ impl<Window: WindowMethods> IOCompositor<Window> {
                 }
             }
 
+            (Msg::Dispatch(func), _) => {
+                func();
```

Comment:
Could we add a comment here like:

```
// The functions sent here right now are really dumb, so they can't panic. But if we start running more complex code here, we should really catch panic here.
```

That should be enough for now. Also, how about renaming the `Dispatch` message to `DispatchForWebGLContextCreation` (or something like that)? I don't think we want this to become a general API.

I filled https://github.com/emilio/rust-offscreen-rendering-context/issues/72, somewhat related to error handling here.